pkg/sqlfs/hive_writer.go

```diff
@@ -79,7 +77,7 @@ func uploadCSVFile(csv *os.File, db *sql.DB, hivePath, table, user, passwd strin
 	}
```

Comment:
Add `defer removeHDFSDir(w.hdfsPath(), hdfsEnv)` at L67 to fix #1345 , maybe we can mrege this PR and I can do that in the next PR.